Resources/views/page_header_logo.html.twig

```diff
@@ -1,3 +1,3 @@
-<div class="span8">
-    <a href="{{ path( ezpublish.rootLocation ) }}" title="" class="logo"><img src="{{ asset('bundles/ezdemo/images/logo-ez.png') }}" alt="" /></a>
+<div class="col-md-8 col-xs-6 col-sm-6">
```

Comment:
`col-xs-6 col-md-8` is more than enough. `col-xs-6` is considered first when rendering and when requirements for `col-md-8` are met then new CSS styles applies.